app/data/jsonSchemaForm/expensesPaymentsTrackerSchema.ts

```diff
@@ -0,0 +1,69 @@
+export const expensesPaymentsTrackerSchema = {
+  $schema: "http://json-schema.org/draft-07/schema",
+  type: "object",
+  required: [
+    "eligibleExpensesToDate",
+    "grossPaymentsToDate",
+    "holdbackAmountToDate",
+    "netPaymentsToDate",
+  ],
+  properties: {
+    eligibleExpensesToDate: {
+      title: "Total Eligible Expenses to Date",
+      type: "number",
+    },
+    grossPaymentsToDate: {
+      title: "Total Gross Payment Amount to Date",
+      type: "number",
+    },
+    holdbackAmountToDate: {
+      title: "Total Holdback Amount to Date",
+      type: "number",
+    },
+    netPaymentsToDate: {
+      title: "Total Net Payment Amount to Date",
+      type: "number",
+    },
+  },
+};
+
+export const expensesPaymentsTrackerUiSchema = {
+  "ui:order": [
+    "eligibleExpensesToDate",
+    "grossPaymentsToDate",
+    "holdbackAmountToDate",
+    "netPaymentsToDate",
+  ],
+  eligibleExpensesToDate: {
+    "ui:widget": "CalculatedValueWidget",
+    calculatedValueFormContextProperty: "calculatedEligibleExpensesToDate",
+    hideOptional: true,
+    isMoney: true,
+    message:
+      "ask designers what message they want here when value isn't yet calculated",
```

Comment:
@nanyangpro, @suhafa , what would you like shown if we don't have enough information entered to do the calculation?